src/config.rs

```diff
@@ -277,6 +280,7 @@ pub fn construct_config(pc: ProxyConfig) -> Result<Config, Error> {
         inbound_plaintext_addr: SocketAddr::new(IpAddr::V6(Ipv6Addr::UNSPECIFIED), 15006),
         outbound_addr: SocketAddr::new(IpAddr::V6(Ipv6Addr::UNSPECIFIED), 15001),
 
+        network: parse(NETWORK)?.unwrap_or_else(|| "defaultnw".to_string()),
```

Comment:
**suggestion:** "defaultnw" as a hardcoded str is used quite a few times in different places. consider making it a `const`

Reply:
updated default network to `""` in https://github.com/istio/ztunnel/pull/486/commits/54631ccc0fa4a2fd54db23cb60e3f629fc30b350 cc @GregHanson